custom-crawls/README.md

```diff
@@ -0,0 +1,42 @@
+# Tutorial: Custom Crawlers
+
+_Note: This tutorial is a work in progress. Please add your feedback to [datatogether/learning](https://github.com/datatogether/learning/issues)!_
+
+## Prerequisites
+
+* You care about a dataset that exists on the web and downloading the data could be benifit from some automation of downloading
+
+## Learning Objectives
+
+After going through this tutorial you will know
+
+* What a custom crawler is and why some websites need one
+* What should your custom crawler extract from a webpage?
+* How to write a custom crawler that works with DataTogether
+
+## Key Concepts
+
+* Custom Crawler:  An auotmated way to download data and prepare it for upload into the DataTogether network. This is usually a script file that is written specifically for a dataset.
+* Morph.io: An online service that automates and saves user created scripts.
+* Archivertools:  An Python package to aid in accessing Morph.io and DataTogether APIs using an Archiver class. This package also contains some common scraping functions. Currently written in Python 3.
```

Comment:
delete extra space
s/DataTogether/Data Together

wondering if you want to link to archivertools? (And how is this package published?)